include/libdnf/comps/group/group.hpp

```diff
@@ -31,10 +31,11 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace libdnf::comps {
 
+
 struct GroupId {
 public:
     GroupId() = default;
-    explicit GroupId(int id);
+    explicit GroupId(int id) : id(id) {}
 
     bool operator==(const GroupId & other) const noexcept { return id == other.id; };
     bool operator!=(const GroupId & other) const noexcept { return id != other.id; };
```

Comment:
Two more occurrences of a semicolon at the end above. Otherwise looks good.

Reply:
I didn't add them during this PR and I wasn't sure if it's best practice to not put them there or if it was just a glitch with the clang-format, because it's inconsistent in other parts of code as well (I originally took it from https://github.com/rpm-software-management/libdnf/blob/dnf-5-devel/include/libdnf/rpm/package.hpp#L52 ). But I will remove them.